my-app/src/redux/game/actions.js

```diff
@@ -0,0 +1,4 @@
+export const actions = {
+  handleClick: 'HANDLECLICK',
+  jumpTo: 'JUMPTO'
```

Comment:
Remember to add namespaces to these action names:
```
handleClick: '@@GAME/HANDLECLICK'
```
This way we will improve debuggability using redux devtools